src/views/AccountPage.vue

```diff
@@ -1,68 +1,69 @@
+          <template v-for="operation in this.operations" :key="operation">
+            <OperationForm v-if="operation.editing" @update-operation-list="getAccountOperation" :accountId="this.accountId" :operation="operation"/>
+            <div v-else class="operation storedOperation">
+              <div class="date col-2 offset-1">
+                <div>{{ $d(this.getDayAsDate(operation.day), "day") }}</div>
+              </div>
+              <div class="col-8"></div>
+              <div class="category col-3 offset-1">
+                {{ this.getCategoryById(operation.categoryId)?.name ?? $t("UNKNOWN_CATEGORY") }}
+              </div>
+              <div class="amount col-3 offset-2">
+                {{ this.getEurosAmount(operation.amount) }} €
+              </div>
+              <div class="action col-1">
+                <button class="btn fas fa-pen" v-on:click="setAsEditing(operation)" :title="$t('EDIT')"/>
+                <button class="btn fas fa-trash" v-on:click="deleteOperation(operation)" :title="$t('DELETE')"/>
+              </div>
+              <div class="memo col-3 offset-1">{{ operation.memo }}</div>
             </div>
-            <div class="action col-1 offset-1">
-              <button class="btn fas fa-pen" v-on:click="setAsEditing(operation)" :title="$t('EDIT')"/>
-              <button class="btn fas fa-trash" v-on:click="deleteOperation(operation)" :title="$t('DELETE')"/>
-            </div>
-            <div class="memo col-3 offset-1">{{ operation.memo }}</div>
-          </div>
-        </template>
-        <ul id="actionsList" class="list-group list-group-horizontal d-flex justify-content-center">
-            <li><button v-on:click="goHomePage" class="btn fas fa-home" :title="$t('HOME')"/></li>
-            <li><button class="btn fas fa-chart-line disabled" :title="$t('GRAPH_AND_REPORT')"/></li>
-            <li><button class="btn fas fa-cog disabled" :title="$t('PREFERENCES')"/></li>
-            <li><button v-on:click="logout" class="btn fas fa-sign-out-alt" :title="$t('LOGOUT')"/></li>
-          </ul>
+          </template>
+          <div class="rectangleBottom"/>
+        </div>
+      </div>
+      <div class="btnInAccountPage">
+        <BtnMenu />
       </div>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue'
-import router, { RouterPages, redirectToLoginPageIfNotLogged } from '@/router'
+import { redirectToLoginPageIfNotLogged } from '@/router'
 import { Account, Category, Operation } from '@/model/model'
 import Time from '@/utils/Time'
 import StoreHandler from '@/store/StoreHandler'
 import OperationService from '@/services/OperationService'
 import AccountService from '@/services/AccountService'
 import OperationForm from '@/components/forms/OperationForm.vue'
 import Utils from '@/utils/Utils'
-import PersonService from '@/services/PersonService'
+import BtnMenu from '@/components/ButtonMenu.vue'
```

Comment:
What would you think of NavMenu instead of BtnMenu ?

Reply:
Why not